revision-static-assets/README.md

```diff
@@ -0,0 +1,86 @@
+# Revision static assets
+
+## Functionality
+
+Give files a unique filename based on their contents and update references to them. A hash is generated based on a file's contents and added to its filename. This ensures a filename only changes when a file's contents change.
+Revisioning allows browsers to aggresively cache these assets by *setting their cache headers to never expire*. 
+
+
+## Usage
+
+### Install dependencies
+
+All dependencies needed to run this recipe are pre-configured in [`package.json`](package.json). You just need to install these:
+
+```bash
+npm install
+```
+
+Note: it's smart to update to the latest versions of these dependencies.
+
+
+### Configure working directory and file patterns
```

Comment:
Im missing the file pattern part here, while it is in the title. What are static assets? Now its done base on the working dir and some patterns. Maybe do it just on the directory, copy all static assets to a static dir and revhash everything in that directory regardless of filetype? Or put the patterns in a variable and describe in the readme how to adjust file patterns?

Reply:
I'm adding another note and adding an annotation to the revision script `gulp.src` lines.